training/production-deployments.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Production Deployments
+summary: Learn how we recommend deploying CockroachDB to production.
+toc: false
+sidebar_data: sidebar-data-training.json
+---
+
+<div id="toc"></div>
+
+## Presentation
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vSIZFCL0gZ7f16BR4RUR0v67p5oq4CIguGDBhiq2_AV8QYpj_yrl-D8UTHI8NR6akR0l_get3jXqNTx/embed?start=false&loop=false" frameborder="0" width="756" height="454" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```

Comment:
Feels like the I/O-related `sysctl` options @petermattis discovered should be mentioned.